solutions/quickstart/main.tf

```diff
@@ -0,0 +1,320 @@
+}
+
+##############################################################################################################
+# watsonx Orchestrate
+##############################################################################################################
+
+module "watsonx_orchestrate" {
+  source                   = "terraform-ibm-modules/watsonx-orchestrate/ibm"
+  version                  = "1.11.6"
+  region                   = var.region
+  resource_group_id        = module.resource_group.resource_group_id
+  watsonx_orchestrate_name = "${local.prefix}orchestrate"
+  plan                     = local.watson_plan["orchestrate"]
+  resource_tags            = var.resource_tags
+}
+
+##############################################################################################################
+# Elastic search
+##############################################################################################################
+
+module "icd_elasticsearch" {
+  source                   = "terraform-ibm-modules/icd-elasticsearch/ibm"
+  version                  = "2.4.5"
+  resource_group_id        = module.resource_group.resource_group_id
+  name                     = "${local.prefix}data-store"
+  region                   = var.region
+  plan                     = "enterprise"
+  elasticsearch_version    = "8.12"
+  tags                     = var.resource_tags
+  service_endpoints        = "private"
+  member_host_flavor       = "multitenant"
+  deletion_protection      = false
+  service_credential_names = local.es_credentials
+}
+
+##############################################################################################################
+# Container Registry
+##############################################################################################################
+
+module "icr_namespace" {
+  source            = "terraform-ibm-modules/container-registry/ibm"
+  version           = "2.2.1"
+  resource_group_id = module.resource_group.resource_group_id
+  namespace_name    = "${local.prefix}namespace"
+}
+
+##############################################################################################################
+# Code Engine - Project, Secret, Build, Application
+##############################################################################################################
+
+locals {
+  env_vars = [
+    {
+      type  = "literal"
+      name  = "WATSONX_AI_APIKEY"
+      value = var.ibmcloud_api_key
+    },
+    {
+      type  = "literal"
+      name  = "WATSONX_PROJECT_ID"
+      value = module.watsonx_ai.watsonx_ai_project_id
+    },
+    {
+      type  = "literal"
+      name  = "ENABLE_WXASST"
+      value = "true"
+    },
+    {
+      type  = "literal"
+      name  = "WXASST_REGION"
+      value = var.region
+    },
+    {
+      type  = "literal"
+      name  = "ENABLE_RAG_LLM"
+      value = "true"
+    }
+  ]
+  cr_region    = split("-", var.region)[0]
```

Comment:
This logic will not work for all regions (see mapping [here](https://github.com/terraform-ibm-modules/terraform-ibm-container-registry/blob/c6e0364f38c0b7504881c87c01de2620c6f44cab/modules/endpoint/main.tf#L8-L24))
We already have a module to handle returning the endpoint. See https://github.com/terraform-ibm-modules/terraform-ibm-container-registry/tree/main/modules/endpoint